src/baustein.js

```diff
@@ -366,6 +394,16 @@ export function isComponent(component) {
  */
 export function handleEvent(event, componentsChain) {
 
+    if (handlingEvent) {
+        pendingEvents.push(arguments);
+        return;
+    }
+
+    handlingEvent = true;
+
+    ensureEventHasMethod(event, 'stopPropagation', 'propagationStopped');
+    ensureEventHasMethod(event, 'preventDefault', 'defaultPrevented');
```

Comment:
Maybe we should start looking at using real events? (https://developer.mozilla.org/en-US/docs/Web/API/Event/Event)

Or, is there a benefit to custom ones?